.github/workflows/build.yml

```
@@ -83,20 +83,31 @@ jobs:
+        run: |
+          mkdir -p ./artifact
+          printf '{
+            "is_pr": false,
+            "pr_id": -1
+          }' >> ./artifact/artifact.json
+
+      - name: Save pull request artifact
         if: github.event_name == 'pull_request'
         env:
           PR_ID: ${{ github.event.number }}
         run: |
```

actionlint

shellcheck reported issue in this script: SC2016:info:2:8: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)